<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyEsp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="text_browser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edit_ip">
        <property name="placeholderText">
         <string>ESP32 IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_path">
        <property name="text">
         <string>指定工程路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDEPLOY">
    <property name="title">
     <string>TOOLS</string>
    </property>
    <addaction name="action_init_hotload"/>
    <addaction name="action_clean_up"/>
   </widget>
   <widget class="QMenu" name="menuABOUT">
    <property name="title">
     <string>ABOUT</string>
    </property>
    <addaction name="action_about_1zlab"/>
   </widget>
   <addaction name="menuDEPLOY"/>
   <addaction name="menuABOUT"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_init_hotload">
   <property name="text">
    <string>deploy hotload to esp32</string>
   </property>
  </action>
  <action name="action_about_1zlab">
   <property name="text">
    <string>1ZLAB</string>
   </property>
  </action>
  <action name="action_clean_up">
   <property name="text">
    <string>clean up esp32</string>
   </property>
  </action>
  <action name="actionWIFI">
   <property name="text">
    <string>WIFI</string>
   </property>
  </action>
  <action name="actionSerial_Port">
   <property name="text">
    <string>Serial Port</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
